ui/Screen/Lock.svelte

```diff
@@ -43,11 +43,12 @@
 
 <div class="lock">
   <Emoji emoji={'🚪'} size="huge" style="margin-bottom: 0.75rem;" />
-  <h3>Unlock the app</h3>
+  <h3>Knock, knock</h3>
+  <p>Enter your passphrase to unlock your Radicle secret key.</p>
```

Comment:
```suggestion
    <p style="margin-top: 0.5rem; color: var(--color-foreground-level-6);">Enter your passphrase to unlock your Radicle secret key.</p>

```

Reply:
Also just looks broken

<img width="501" alt="Screen Shot 2020-11-23 at 14 34 45" src="https://user-images.githubusercontent.com/2326909/99968147-0ca02480-2d99-11eb-95d6-c8ce51f70c16.png">

Reply:
I'd rather say what entering the password does, rather than what it technically does. It unlocks the app. The user doesn't know about a 🔮 secret 🗝 key 🌝

Like my question when I read this text is: Once I enter my password here and I've unlocked my secret key, how do I use that key to unlock the app? Why even mention it? Your password unlocks the app, that's it.

Reply:
My question as a user would then be: why is the app locked?

Reply:
I see both sides. While the passphrase unlocks the app, it is also explicitly associated with a user's keys. Therefore, I think they *should* know about the keys. But, we can make this statement more user friendly.

Here is my iteration taking into consideration the fact that we will have `Enter your passphrase` in the input field:

```suggestion
  <p>Unlock the app by granting access to your keys with your passphrase</p>
```

Reply:
What I gauged from @cloudhead was that the passphrase unlocks the app sure, but it also gives you publishing access to the Radicle Network, so it's not exactly just an 'app' passphrase, since you also add it when you push. Since the passphrase *does* decrypt a keypair so they can be used by the user, I think they should associate it with keys.

Here's my iteration:

' Decrypt your keys with your passphrase to unlock the app.`

> > * Add styling for `Lock` and `Manage Remotes` (@juliendonck and/or @brandonhaslegs ?)
> 
> Addressed this in my review
> 
> @abbey-titcomb how are you expecting the other work to be addressed?

Will update the TODOs once we resolve the conversations and ask people to implement from there.

Reply:
I like where this is going, I'd propose an iteration on the above:

"Unlock the app by granting access to your Radicle secret key"
[ Enter your passphrase ] [Unlock]

Reply:
@brandonhaslegs - all good with this?

Reply:
Ah cool, I like the update. Now the placeholder is not repeating information. Cool!